src/core/shape_inference/include/fft_common_validation.hpp

```diff
@@ -10,23 +10,23 @@
 namespace ov {
 namespace op {
 namespace util {
-namespace rfft_common_validation {
-enum class RFFTKind { Forward, Inverse };
+namespace fft_common_validation {
+enum class FFTKind { RealInput, ComplexInput };
```

Comment:
Future improvement, use c-style `enum` or some `constexpr` values for this FFT kind instead? 
Then 
```const size_t min_rank = (fft_kind == FFTKind::RealInput) ? 1 : 2;``` 
will be not required as FFT kind will have correct value.

Reply:
Added as a part of https://github.com/openvinotoolkit/openvino/pull/19463.